infinispan-remote/cache-admin-api/README.adoc

```diff
@@ -9,7 +9,7 @@ on Infinispan servers.
 . Build and run the Hot Rod Java client as follows:
 
 ----
-$ mvn clean package
```

Comment:
this is a README file and not part of the docs. is the attribute going to be replaced?